heron/api/src/java/com/twitter/heron/api/serializer/KryoSerializer.java

```diff
@@ -1,51 +0,0 @@
-// Copyright 2016 Twitter. All rights reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package com.twitter.heron.api.serializer;
-
-import java.util.Map;
-
-import com.esotericsoftware.kryo.Kryo;
-import com.esotericsoftware.kryo.io.Input;
-import com.esotericsoftware.kryo.io.Output;
-
-public class KryoSerializer implements IPluggableSerializer {
```

Comment:
Nobody used this before, because everyone uses `heron-storm` lib and therefore `HeronPluggableSerializerDelegate`.

We should dive into it again when people will start to use `heron-api` directly (if this will happen), but for now I guess it's totally fine to remove this class.